R/SVG.R

```diff
@@ -9,6 +9,8 @@
 #' @param pointsize default point size.
 #' @param standalone Produce a standalone svg file? If \code{FALSE}, omits
 #'   xml header and default namespace.
+#' @param overwrite_page Overwrite the current page when a new page is created.
```

Comment:
I think it does, as this option doesn't change the closing `</svg>` behavior.  This It starts re-writing at the start of the file when a new page is called (See change at [L254](https://github.com/hadley/svglite/pull/53/files#diff-fab7ff915f02697381cc9ca70e199259R254).  I implemented this so I could watch a single file as I iterated on a plot.  However, #46 provides a better way to do this, so I will remove this option.